doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,226 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
-```
-
-There are more convenience functions such as ``value_or`` which provides the value or an alternative specified by the
-user. These can be found in [``expected.hpp``](../../../iceoryx_utils/include/iceoryx_utils/cxx/expected.hpp).
-
-## Using the C++ API
-
-Armed with the terminology and functional concepts, we can start to use the API to send and receive data. This
-involves setting up the runtime in each application, creating publishers in applications that need to send data and
-subscribers in applications that want to receive said data. An application can have both, publishers and subscribers.
-It can even send data to itself, but this usually makes little sense.
-
-### Initialize the runtime
-
-Create a runtime with a unique name among all applications for each application
+Before we get into more details let's start with a simple example:
 
+We need to create a runtime with a unique name along all applications for each application.
 ```cpp
 iox::runtime::PoshRuntime::initRuntime("some_unique_name");
 ```
-
-Now this application is ready to communicate with the middleware daemon RouDi.
-
-### Defining a topic
-
-We need to define a data type we can send, which can be any struct or class or even a plain type, such as an int.
-
+Now this application is ready to communicate with RouDi and we can define the data type we want to send.
 ```cpp
 struct CounterTopic
 {
-    CounterTopic(uint32_t counter = 0U)
-        : counter(counter)
-    {
-    }
-
     uint32_t counter;
 };
 ```
-
-The topic type must be default- and copy-constructible when the typed API is used. Using the untyped API imposes no
-such restriction, it just has to be possible to construct the data type at a given memory location.
-
-### Typed API
-
-We now demonstrate how to send and receive data with the typed API. This API is mainly used when iceroryx is used
-stand-alone, i.e. not integrated into a third party framework, and provides a high level of type safety combined
-with [RAII](https://en.cppreference.com/w/cpp/language/raii).
-
-#### Creating a publisher
-
-We create a publisher that offers our CounterTopic.
-
+Then we create a publisher that offers our CounterTopic.
 ```cpp
 iox::popo::Publisher<CounterTopic> publisher({"Group", "Instance", "CounterTopic"});
-publisher.offer();
 ```
-
-Note that it suffices to set the first two identifiers (Group and Instance) to some default values for all topics.
-
-#### Sending data
-
-Now we can use the publisher to send the data in various ways.
-
-##### 1. Loan and assign
-
+Now we can use the publisher to send the data.
 ```cpp
 auto result = publisher.loan();
-if (!result.has_error())
+if(!result.has_error())
 {
     auto& sample = result.value();
-    sample->counter = 73;
+    sample->counter = 30;
     sample.publish();
 }
 else
 {
     // handle the error
 }
 ```
+Here ``result`` is an ``expected`` and hence we may get an error which we have to handle. This can happen if we try 
```

Comment:
Please add the hint that you have to handle the error somehow since the expected class has the nodiscard keyword attached. Otherwise you will get a warning or and error when you build in strict mode.

If you do not want to handle the error you can use `IOX_DISCARD_RESULT(myMethodThatReturnsAnExpected())`